crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -138,6 +142,14 @@ impl RenderDevice {
         desc: &wgpu::TextureDescriptor,
         data: &[u8],
     ) -> Texture {
```

Comment:
This should probably return `Result<Texture, SomeError>` instead of logging an error (missable depending on logging config) and panicking.

Reply:
How should the error be ultimately handled, once it's bubbled up, should the app crash with error message, just not display any image and warn, or what?

Reply:
That's up to the caller to decide. In the case of Bevy first party crates, it's probably best to handle the error properly and retry with the appropriate size, and panic if truly unrecoverable.